### PR TITLE
Make AU eRequesting ServiceRequest abstract (FHIR-46813) and move bodySite from derived profiles to the parent profile (FHIR-47168)

### PR DESCRIPTION
**Pull request by johnscarter**
Fixes for:
1. [FHIR-46813](https://jira.hl7.org/browse/FHIR-46813) Make base request abstract
2. [FHIR-47168 ](https://jira.hl7.org/browse/FHIR-47168) bodySite profiles in pathology, imaging profiles can be in the parent profile

### CODE REVIEW

File: fsh-generated/fsh-index.txt [hunk 1]
```diff
@@ -3,8 +3,9 @@ ActorDefinition-au-erequesting-actor-filler.json             au-erequesting-acto
...
+StructureDefinition-au-erequesting-servicerequest-imag.json  AUeRequestingImagingRequest         Profile   au-erequesting-servicerequest-imag.fsh                    1 - 24
+StructureDefinition-au-erequesting-servicerequest-path.json  AUeRequestingPathologyRequest       Profile   au-erequesting-servicerequest-path.fsh                    1 - 23
+StructureDefinition-au-erequesting-servicerequest.json       AUeRequestingServiceRequest         Profile   au-erequesting-servicerequest.fsh                         1 - 204
 StructureDefinition-au-erequesting-task.json                 AUeRequestingTask                   Profile   au-erequesting-task.fsh                                   1 - 40
-ValueSet-ranzcr-radiology-referral.json                      RANZCRRadiologyReferral             ValueSet  valuesets\au-erequesting-ranzcr-radiology-procedures.fsh  1 - 527
\ No newline at end of file
+ValueSet-au-erequesting-task-status.json                     AUeRequestingTaskStatus             ValueSet  valuesets\au-erequesting-task-status.fsh                  1 - 21
```
Comment (michael-wilson-au) [inline fsh-generated/fsh-index.txt:10]: There shouldn't be AUeRequestingTaskStatus changes in this PR.

File: fsh-generated/resources/ImplementationGuide-hl7.fhir.au.ereq.json
Comment (michael-wilson-au) [inline fsh-generated/resources/ImplementationGuide-hl7.fhir.au.ereq.json:1]: Remove AUeRequestingTaskStatus changes from this PR

File: fsh-generated/includes/fsh-link-references.md
Comment (michael-wilson-au) [inline fsh-generated/includes/fsh-link-references.md:1]: Remove AUeRequestingTaskStatus changes from this PR

File: fsh-generated/fsh-index.json [hunk 2]
```diff
@@ -63,12 +63,20 @@
     "startLine": 1,
     "endLine": 40
   },
+  {
```
Comment (michael-wilson-au) [inline fsh-generated/fsh-index.json:66]: Remove AUeRequestingTaskStatus changes from this PR

File: fsh-generated/resources/ValueSet-au-erequesting-task-status.json
Comment (michael-wilson-au) [inline fsh-generated/resources/ValueSet-au-erequesting-task-status.json:1]: Remove AUeRequestingTaskStatus changes from this PR

File: fsh-generated/resources/ValueSet-ranzcr-radiology-referral.json [hunk 3]
```diff
@@ -19,9 +19,9 @@
...
...h Terminology Standards Development Organisation (IHTSDO), and distributed by agreement between IHTSDO and HL7. Implementer use of SNOMED CT is not covered by this agreement\nThe SNOMED International IPS Terminology is distributed by International Health Terminology Standards Development Organisation, trading as SNOMED International, and is subject the terms of the [Creative Commons Attribution 4.0 International Public License](https://creativecommons.org/licenses/by/4.0/). For more information, see [SNOMED IPS Terminology](https://www.snomed.org/snomed-ct/Other-SNOMED-products/international-patient-summary-terminology)\n\r\nThe HL7 International IPS implementation guides incorporate SNOMED CT®, used by permission of the International Health Terminology Standards Development Organisation, trading as SNOMED International. SNOMED CT was originally created by the College of American Pathologists. SNOMED CT is a registered trademark of the International Health Terminology Standards Development Organisation, all rights reserved. Implementers of SNOMED CT should review [usage terms](http://www.snomed.org/snomed-ct/get-snomed-ct) or directly contact SNOMED International: info@snomed.org",
```
Comment (michael-wilson-au) [inline fsh-generated/resources/ValueSet-ranzcr-radiology-referral.json:24]: Is there a difference in this copyright statement?

**Review by michael-wilson-au — CHANGES_REQUESTED** [review]

File: fsh-generated/fsh-index.txt (hunk 1 above)
Comment (johnscarter) [reply fsh-generated/fsh-index.txt:10]: these changes are the result of auto-generated files, no change to AUeRequestingTaskStatus was made

File: fsh-generated/resources/ImplementationGuide-hl7.fhir.au.ereq.json
Comment (johnscarter) [reply fsh-generated/resources/ImplementationGuide-hl7.fhir.au.ereq.json:1]: as above

File: fsh-generated/includes/fsh-link-references.md
Comment (johnscarter) [reply fsh-generated/includes/fsh-link-references.md:1]: as above

File: fsh-generated/fsh-index.json (hunk 2 above)
Comment (johnscarter) [reply fsh-generated/fsh-index.json:66]: as above

**Comment by johnscarter** [comment]
I resolved a merge conflict with changes.md and recommitted. Today, I did a commit on just the files I'd actually changed, then did the IGPublisher build, and then committed the auto-generated files separately. That way, if needed to back out the auto-generated files, it will be easier.

File: input/fsh/au-erequesting-task.fsh [hunk 4]
```diff
@@ -15,6 +15,7 @@ Description: "This profile sets minimum expectations for a Task resource that is
 
 
 * status MS
+* status from http://hl7.org.au/fhir/ereq/ValueSet/au-erequesting-task-status (required)
```
Comment (dbojicic) [inline input/fsh/au-erequesting-task.fsh:18]: This change is not in scope of this PR, please remove.

File: input/fsh/au-erequesting-servicerequest.fsh [hunk 5]
```diff
@@ -189,6 +191,19 @@ Description: "This profile sets minimum expectations for a ServiceRequest resour
 * reasonCode ^extension[http://hl7.org/fhir/StructureDefinition/obligation][4].extension[actor][0].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-patientaccess"
```
Comment (dbojicic) [inline input/fsh/au-erequesting-servicerequest.fsh:193]: Commenting here to get an inline comment. 

Change to remove AU eRequesting ServiceRequest as supported profile from CapabilityStatements is missing.

File: input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md [hunk 6]
```diff
@@ -1,8 +1,7 @@
 <p class="request-for-feedback">Input is requested on the appropriateness of allowing Missing Data or Suppressed Data for all elements. Please comment by raising <a href="https://jira.hl7.org/projects/FHIR/issues">HL7 Jira Issues</a>.</p>
 
 ### Profile specific implementation guidance
-- This profile provides a shared base that is common across requests for specific domains. Where a profile is defined for a specific domain that profile **SHALL** be used:
+- This abstract (non-implementable) profile provides a shared base that is common across requests for specific domains. Note the following domain-specific requests, associated profiles, and usage requirements:
   - pathology requests **SHALL** use [AU eRequesting Pathology Request](StructureDefinition-au-erequesting-servicerequest-path.html) profile
   - imaging requests **SHALL** use [AU eRequesting Imaging Request](StructureDefinition-au-erequesting-servicerequest-imag.html) profile
```
Comment (dbojicic) [inline input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md:6]: I agree with changes to reflect the abstract nature of this profile. I suggest a slight change:

> This abstract profile provides a shared base that is common across requests for specific domains. It is not intended for direct implementation. Where a profile is defined for a specific domain that profile **SHALL** be used:
> - pathology requests **SHALL** use [AU eRequesting Pathology Request](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/branches/master/StructureDefinition-au-erequesting-servicerequest-path.html) profile
> - imaging requests **SHALL** use [AU eRequesting Imaging Request](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/branches/master/StructureDefinition-au-erequesting-servicerequest-imag.html) profile

Suggestions are welcome! @michael-wilson-au, any thoughts?

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see comments.

File: input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md (hunk 6 above)
Comment (michael-wilson-au) [reply input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md:6]: I'm happy with the suggested change from @dbojicic

**Comment by michael-wilson-au** [comment]
> I resolved a merge conflict with changes.md and recommitted. Today, I did a commit on just the files I'd actually changed, then did the IGPublisher build, and then committed the auto-generated files separately. That way, if needed to back out the auto-generated files, it will be easier.

@dbojicic has now removed the fsh-geneated folder from master so this branch may need to be updated to remove those files.

File: input/fsh/au-erequesting-servicerequest.fsh (hunk 5 above)
Comment (johnscarter) [reply input/fsh/au-erequesting-servicerequest.fsh:193]: done

**Review by johnscarter — COMMENTED** [review]
ready for review

**Review by dbojicic — APPROVED** [review]

File: input/fsh/au-erequesting-task.fsh
Comment (michael-wilson-au) [inline input/fsh/au-erequesting-task.fsh:1]: Is this still a merge issue?  There shouldn't be any Task changes in this PR.

Comment (johnscarter) [reply input/fsh/au-erequesting-task.fsh:1]: I copied teh contents of this file from the repo Main branch.

Comment (johnscarter) [reply input/fsh/au-erequesting-task.fsh:1]: So hopefully now it's in the correct state!

**Review by michael-wilson-au — APPROVED** [review]

**Pull request merged**